src/components/CommentCard.jsx

```diff
@@ -1,14 +1,31 @@
 import "../styles/CommentCard.css"
+import { deleteComment } from "../utils/fetches"
 
-const CommentCard = ({ comment }) => {
+const CommentCard = ({
+  comment,
+  username,
+  refreshComments,
+  setRefreshComments,
+}) => {
   const date = new Date(comment.created_at)
-  comment={...comment, created_at: date.toUTCString()}
+  comment = { ...comment, created_at: date.toUTCString() }
```

Comment:
Check your back end, is it expecting a time stamp or does the database do that for you...